Changes

```diff
@@ -100,6 +100,9 @@ Next version (4.05.0):
 - clarify ocamldoc text parsing error messages
   (Gabriel Scherer)
 
+- Make "-open" command line flag accept a module path (not a module name)
```

Comment:
Giving the issue numbers is important for traceability. Your entry should start with `PR#7137, GPR#960`, and given the entry ordering (by MPR first, then by GPR), it should be the first in the section.